core/shared/nql-map-key-values/README.md

```diff
@@ -0,0 +1,26 @@
+# nql-map-key-values
```

Comment:
@allouis this piece feels like it naturally would belong in [mongo-utils](https://github.com/NexesJS/mongo-utils/). Remember even mentioning this in the original PR - https://github.com/TryGhost/Ghost/pull/10944#discussion_r309084174 :wink: Was there some specific reason you wanted to keep it in the core?

Reply:
Going to raise an issue in mongo-utils for this